packages/components/src/draggable/index.native.js

```diff
@@ -158,7 +174,10 @@ const DraggableTrigger = ( {
 			}
 		},
 		onEnd: () => {
-			isDragging.value = false;
+			if ( ! isPanActive.value ) {
```

Comment:
It adds this check because while removing extra fingers on the screen it would call the `onEnd` from the `LongPress` gesture. This way it only set's the `isDragging` flag to false if the `Pan` gesture is not active.